bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/profiles/TriggerProfileType.java

```diff
@@ -23,13 +22,8 @@
 public interface TriggerProfileType extends ProfileType {
 
     /**
-     * Constant for matching *ANY* channel type.
-     */
-    Collection<ChannelTypeUID> ANY_CHANNEL_TYPE = new ArrayList<>(0);
-
-    /**
      *
-     * @return a collection of ChannelTypeUIDs or {@link #ANY_CHANNEL_TYPE}.
+     * @return a collection of ChannelTypeUIDs (may be empty all are supported).
```

Comment:
I am not a native speaker but isn't there a "if" missing (may be empty if all are supported)?